src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -81,7 +80,76 @@ class PdlClient(
         }
     }
 
+    suspend fun aktorId(
+        callId: String,
+        personIdentNumber: PersonIdentNumber,
+    ): AktorId {
+        val token = azureAdV2Client.getSystemToken(pdlClientId)
+            ?: throw RuntimeException("Failed to send request to PDL: No token was found")
+        return identer(
+            callId = callId,
+            ident = personIdentNumber.value,
+            token = token,
+        )?.aktorId()
+            ?: throw RuntimeException("PDL returned empty navn for given fnr")
+    }
+
+    suspend fun identer(
+        callId: String,
+        ident: String,
+        token: AzureAdV2Token,
+    ): PdlHentIdenter? {
+        val request = PdlHentIdenterRequest(
+            query = getPdlQuery("/pdl/hentIdenter.graphql"),
+            variables = PdlHentIdenterRequestVariables(
+                ident = ident,
+                historikk = false,
+                grupper = listOf(
+                    IdentType.AKTORID.name,
+                    IdentType.FOLKEREGISTERIDENT.name
+                )
+            )
+        )
+
+        val response: HttpResponse = httpClient.post(pdlUrl) {
+            body = request
+            header(HttpHeaders.ContentType, "application/json")
+            header(HttpHeaders.Authorization, bearerHeader(token.accessToken))
+            header(TEMA_HEADER, ALLE_TEMA_HEADERVERDI)
+            header(NAV_CALL_ID_HEADER, callId)
+            header(IDENTER_HEADER, IDENTER_HEADER)
```

Comment:
Dette ble en hack, fordi jeg ikke klarte å differensiere de to ulike requestene i PdlMock.kt.